eclipsePlugin-junit/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: "$rootDir/gradle/checkstyle.gradle"
+
+dependencies {
+  compile project(':eclipsePlugin')
+  testCompile 'junit:junit:4.12'
+  testCompile 'org.mockito:mockito-core:2.7.22'
+}
```

Comment:
My motivation is run my JUnit test by build tool (Gradle for now).
Currenly our build tool cannot run tests in `eclipsePlugin-test` so I added new project.

If we can run tests in `eclipsePlugin-test` just by build tool, then it's enough &amp; better solution, I think.